src/main/java/gov/hhs/cms/bluebutton/fhirstress/utils/BenefitIdMgr.java

```diff
@@ -0,0 +1,100 @@
+package gov.hhs.cms.bluebutton.fhirstress.utils;
+
+public class BenefitIdMgr {
+  protected int currIndex;
+  protected int minIndex;
+  protected int maxIndex;
+  protected String prefix;
+  protected String format;
+
+  /** 
+   * Constructor(s) 
+   *
+   * @param startIndex
+   *  a {@link int} containing the starting index to use for creating benefit IDs
+   * @param minIndex
+   *  a {@link int} containing the minimum index allowed for creating benefit IDs
+   * @param maxIndex
+   *  a {@link int} containing the maximum index allowed for creating benefit IDs
+   * @param prefix
+   *  a {@link String} containing the prefix used for creating benefit IDs
+   * @param format
+   *  a {@link String} containing the string formatter to use for formatting
+   *  indexes in benefit IDs
+   */
+  public BenefitIdMgr(int startIndex, int minIndex, int maxIndex, String prefix, String format) {
+    this.minIndex = minIndex;
+    this.maxIndex = maxIndex;
+    this.prefix = prefix;
+    this.format = format;
+    setCurrIndex(startIndex);
+  }
+
+  /** 
+   * Public Methods 
+   */
+
+  /**
+   * Set the next index to the given value
+   * 
+   * @param index
+   *  a {@link int} value to set the next index to. 
+   */
+  public void setCurrIndex(int index) {
```

Comment:
Why's this method exist? Looks like it's not used in your production code, only in your tests.

Reply:
Made it private as it checks boundary conditions for the constructor and updated tests.